setup.py

```diff
@@ -47,6 +54,89 @@ def load_requirements(path_dir=PATH_ROOT, file_name='base.txt', comment_char='#'
     return reqs
 
 
+def _parse_README_for_badge(text):
```

Comment:
Any chance you could add tests for those functions ?

Reply:
ok i can add test, I was already  thinking about it
do I create a new test file?

Reply:
@tchaton I am having a problem:

I wanted to create a test for the two functions, but since they are inside the setup.py file when i try to import them it runs the setup itself, and in this way it crash...how do you think I should proceed? 

- Move the two functions outside the setup, but where?
- Modifying setup in someway that prevents its run when function from it are imported?